pkg/apis/app/v2/kieapp_types.go

```diff
@@ -478,6 +483,34 @@ type RoleMapperTemplate struct {
 	RoleMapperAuthConfig `json:",inline"`
 }
 
+// ProcessMigrationObject
+type ProcessMigrationObject struct {
+	Image    string                         `json:"image,omitempty"`
+	ImageTag string                         `json:"imageTag,omitempty"`
+	Database ProcessMigrationDatabaseObject `json:"database,omitempty"`
```

Comment:
You can reuse the `DatabaseObject` although up to the CRD `external` is not supported. That would help the refactoring in the next iteration

Reply:
The `crd_validation_test` will fail for "Discrepancy between CRD and Struct". Is there a way we can skip the test for `external`?

Reply:
Right, I didn't think about the crd_validation_test.